index.html

```diff
@@ -24,15 +24,45 @@
   <body>
     <header class="header">
       <nav id="navbar" class="navbar">
-        <a href="">Kennedy Owusu</a>
-        <a id="menu-icon" href=""><img src="./images/nav-icons/nav-icon.png"></a>
+        <a class="navbar-title" href="">Kennedy Owusu</a>
         <ul>
           <li><a class="header-nav" href="">Portfolio</a></li>
           <li><a class="header-nav" href="">About</a></li>
           <li><a class="header-nav" href="">Contact</a></li>
           <li><a class="header-nav" href=""><img src="./images/nav-icons/mail-icon.png"></a></li>
         </ul>
+
+      </nav>
+
+      <nav id="mobile-navbar">
+        
+        <div>
+          <a href="#" class="navbar-title" id="navbar-title">Kennedy Owusu</a>
+
+          <div class="hamburger">
```

Comment:
Good job! I really love the way you animated the hamburger button 👍and you can improve the design by following the exact Figma design in which the mobile menu is taking full height. Please make sure to fix it.

The menu you tried to design
![image](https://user-images.githubusercontent.com/34779928/202004815-eb6048f5-b0ec-4bb4-bab1-2f209b7dad09.png)


Original

![image](https://user-images.githubusercontent.com/34779928/202004831-4fe28fec-55fb-4cae-b467-dcd836219ee9.png)

Reply:
Thank you for your feedback. I have resolved the issue as seen in the image below.

![full-menu](https://user-images.githubusercontent.com/32777408/202012783-969236a4-5a30-4c3f-8935-83194b214389.png)

Reply:
- [ ] I see by the comment that you made the requested change, that's great! 💯 However, I can't see those changes on this repository, please consider pushing them! 🤓 🛠️